docs/en/stack/ml/df-analytics/flightdata-regression.asciidoc

```diff
@@ -0,0 +1,357 @@
+    "Cancelled": false,
+    "FlightTimeMin": 454.6742272195069,
+    "Origin": "Adelaide International Airport",
+    "OriginLocation": {
+      "lat": "-34.945",
+      "lon": "138.531006"
+    },
+    "DestRegion": "SE-BD",
+    "OriginAirportID": "ADL",
+    "OriginRegion": "SE-BD",
+    "DestCityName": "Tokoname",
+    "FlightTimeHour": 7.577903786991782,
+    "FlightDelayMin": 0
+  }
+}
+```
+
+
+{regression-cap} is a supervised machine learning analysis and therefore needs 
+to train on data that contains the ground truth for the `dependent_variable` 
+that we want to predict. In this example, the ground truth is available in each 
+document as the actual value of `FlightDelayMins`. In order to be analyzed, a 
+document must contain at least one field with a supported data type (`numeric`, 
+`boolean`, `text`, `keyword` or `ip`) and must not contain arrays with more than 
+one item.
+
+If your source data consists of some documents that contain a 
+`dependent_variable` and some that do not, the model is trained on the 
+`training_percent` of the documents that contain ground truth. However, 
+predictions are made against all of the data. The current implementation of 
+{reganalysis} supports a single batch analysis for both training and 
+predictions.
+
+
+[[flightdata-regression-model]]
+==== Creating a {regression} model
+
+To predict the number of minutes delayed for each flight:
+. Create a {dfanalytics-job}.
++
+--
+Use the {ref}/put-dfanalytics.html[create {dfanalytics-jobs}] API as you can see 
+in the following example:
+
+[source,console]
+--------------------------------------------------
+PUT _ml/data_frame/analytics/model-flight-delays
+{
+  "source": {
+    "index": [
+      "kibana_sample_data_flights" <1>
+    ],
+    "query": { <2>
+      "range": {
+        "DistanceKilometers": { 
+          "gt": 0
+        }
+      }
+    }
+  },
+  "dest": {
+    "index": "df-flight-delays"  <3>
+  },
+  "analysis": {
+    "regression": {
+      "dependent_variable": "FlightDelayMin",  <4>
+      "training_percent": 90  <5>
+    }
+  },
+  "analyzed_fields": {
+    "includes": [],
+    "excludes": [    <6>
+      "Cancelled",
+      "FlightDelay",
+      "FlightDelayType"
+    ]
+  },
+  "model_memory_limit": "100mb" <7>
+}
+--------------------------------------------------
+// TEST[skip:setup kibana sample data]
+
+<1> The source index to analyze.
+<2> This query removes erroneous data from the analysis to improve its quality.
+<3> The index that will contain the results of the analysis; it will consist of 
+a copy of the source index data where each document is annotated with the 
+results.
+<4> Specifies the continuous variable we want to predict with the {reganalysis}.
+<5> Specifies the proportion of data that is used for training the model. In 
+this example we use 90% of the source data. The training data is randomly 
+selected and provide predictions from the remaining 10%. These percentages are 
+approximate.
```

Comment:
Predictions are made for all of he data, not just the remaining 10%. Therefore should be:

```
Specifies the approximate proportion of data that is used for training the model. In 
this example we randomly select 90% of the source data for training. 
```